databricks/koalas/namespace.py

```diff
@@ -33,20 +33,22 @@
 from databricks.koalas.series import Series
 
 
-def from_pandas(pdf):
-    """Create a Koalas DataFrame from a pandas DataFrame.
+def from_pandas(pdf: Union[pd.DataFrame, pd.Series]) -> Union[ks.Series, ks.DataFrame]:
+    """Create a Koalas DataFrame or Series from a pandas DataFrame or Series.
 
     This is similar to Spark's `DataFrame.createDataFrame()` with pandas DataFrame,
-    but this also picks the index in the given pandas DataFrame.
+    but this also picks the index in the given pandas DataFrame or Series.
```

Comment:
but this also takes Series, and picks the index in the given pandas DataFrame or Series.